src/Compilers/Core/MSBuildTask/Desktop/project.json

```diff
@@ -1,5 +1,7 @@
 {
-  "dependencies": { },
+  "dependencies": {
+    "Roslyn.Microsoft.Build": "0.0.2"
```

Comment:
Yes, but this was an existing package on nuget.org and I see there are bunch of others called "Roslyn." as well - we should clean this up later.